### ISSUE TITLE
Milky Way

### ISSUE THREAD

**Issue by nickh** [nickh/photography#2]
- August is a good time of year (per @jordanmccullough)

---
### Possible Locations
- Dinosaur sculpture farm just east of Half Moon Bay

**Comment by jordanmccullough** [comment]
@nickh, What about during GitHub Summit 2016? Would you care to try some low-light photos at [Garden of the Gods](http://www.gardenofgods.com/)?

**Comment by nickh** [comment]
@jordanmccullough yes, that sounds awesome! From what I've read, I need a wide angle lens, stable tripod, and remote shutter release. Anything else?

**Comment by jordanmccullough** [comment]
If you're wanting to travel light, I'm happy to share my gear:
- 15mm f/2.8 fisheye
- 16mm  f/2.8 wide angle
- 25mm f/2
- 24-70mm f.2.8
- 50mm f/2
- 100mm f/2
- 200mm f/2.8
- Gitzo Mountaineer carbon fiber tripod
- Really Right Stuff B55 ballhead
- Multiple body mount plates for tripod system
- Canon intervalometer

**Comment by nickh** [comment]
Sweet :) I'm going to start an issue for Summit 2016 too, I expect we'll have lots of options!
